topcoffea/modules/corrections.py

```diff
@@ -590,19 +590,18 @@ def LoadTriggerSF(year, ch='2l', flav='em'):
 
 def GetTriggerSF(year, events, lep0, lep1):
   ls=[]
-  for syst in [0,1,2]:
+  for syst in [0,1]:
     #2l
     SF_ee=np.where(events.is_ee==True,LoadTriggerSF(year,ch='2l',flav='ee')[syst](lep0.pt,lep1.pt),1.0)
     SF_em=np.where(events.is_em==True, LoadTriggerSF(year,ch='2l',flav='em')[syst](lep0.pt,lep1.pt),1.0)
     SF_mm=np.where(events.is_mm==True, LoadTriggerSF(year,ch='2l',flav='mm')[syst](lep0.pt,lep1.pt),1.0)
     #3l
     SF_eee=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='eee')[syst](lep0.pt,lep0.eta),1.0)
-    SF_eem=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='eem')[syst](lep0.pt,lep0.eta),1.0)
-    SF_emm=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='emm')[syst](lep0.pt,lep0.eta),1.0)
-    SF_mmm=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='mmm')[syst](lep0.pt,lep0.eta),1.0)
+    SF_eem=np.where(events.is_eem==True,LoadTriggerSF(year,ch='3l',flav='eem')[syst](lep0.pt,lep0.eta),1.0)
+    SF_emm=np.where(events.is_emm==True,LoadTriggerSF(year,ch='3l',flav='emm')[syst](lep0.pt,lep0.eta),1.0)
+    SF_mmm=np.where(events.is_mmm==True,LoadTriggerSF(year,ch='3l',flav='mmm')[syst](lep0.pt,lep0.eta),1.0)
```

Comment:
Yes, I found a bug when I was implementing the changes. Sorry for that, although I hope they don't change much the final result since all 3l SFs are very close to one. Could you update them? I know datacard_maker was giving me problems and I don't remember why.

Reply:
Ok, done. All the changes were <1%.